riptide-core/src/main/java/org/zalando/riptide/RestBuilder.java

```diff
@@ -79,6 +80,11 @@ private URI checkAbsoluteBaseUrl(@Nullable final URI baseUrl) {
         return baseUrl;
     }
 
+    public RestBuilder urlResolution(@Nullable final UrlResolution resolution) {
```

Comment:
The builder allows nulls basically everywhere, I wanted to be consistent. It's nice so clients can read those values directly from their configuration and don't have to double check.